### DISCUSSION TITLE
License Badge rendered as "Other" while it should have been LGPL v3+

### DISCUSSION THREAD

**Discussion by yookoala** [badges/shields#10417] [Q&A] [answered]
The license shield below was generated for the progject: [cangjie/pycangjie](https://gitlab.freedesktop.org/cangjie/pycangjie) on a custom GitLab instance:

[![License Shield](https://github.com/user-attachments/assets/ac1282d3-d56b-40d2-9eff-7d27135c6f46)](https://img.shields.io/gitlab/license/cangjie%2Fpycangjie?gitlab_url=https%3A%2F%2Fgitlab.freedesktop.org)

The project is licensed to LGPL v3+. I think there is some mistake in the license detection mechanism. How was this shield generated? How can I modify the repository to fit the detection?

**Comment by chris48s** [comment, marked as answer]
We report the license GitLab has detected. So in your case,

```
curl "https://gitlab.freedesktop.org/api/v4/projects/cangjie%2Fpycangjie?license=1" | jq .license.name
```

If you think there's a problem with GitLab's license detection, you can raise an issue with GitLab. Usually these types of issues boil down to the text in the license in the user's repo being subtly different in some minor way from the canonical version of the license that the source control platform is comparing against.

Cheers.
